cmd/kcp-config/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v3"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"syscall"
+)
+
+func main() {
+	if len(os.Args) < 3 {
+		fmt.Printf("usage: kcp-config [config-file] [cmd] [args...]\n")
+		os.Exit(1)
+	}
+
+	fname := os.Args[1]
+
+	data, err := ioutil.ReadFile(fname)
+	if err != nil {
+		fmt.Printf("Error reading config file %s: %v\n", fname, err)
+		os.Exit(1)
+	}
+
+	contents := map[string]interface{}{}
```

Comment:
We're passing it as a set of strings to the command, so why is this not `map[string]string`?

How would this allow passing a flag more than once?

Reply:
The yaml document could have boolean or int types that get converted into strings.  We could probably add some type checking to ensure a flat hierarchy (i.e. no nested maps).

You currently can't pass more than one flag.  If we need that, perhaps we could consider a special delimiter for that?